artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/amq/AMQSession.java

```diff
@@ -436,11 +435,11 @@ public ServerSession getCoreSession() {
    public ActiveMQServer getCoreServer() {
       return this.server;
    }
-
+/*
```

Comment:
Why comment this out, the source control maintains history so leaving old dead code in is not needed, you can just review the commit history to see it.

Reply:
doh, forgot it. I think I commented it out back then thinking it might be used somewhere else.
Thanks @tabish121 . Branch updated.